server/models.py

```diff
@@ -1,10 +1,66 @@
-from app import db
+from flask_sqlalchemy import SQLAlchemy
+from flask_marshmallow import Marshmallow
+from marshmallow import Schema, fields
+
 from sqlalchemy.dialects.postgresql import JSON
 
+from app import db
+
+db = SQLAlchemy()
```

Comment:
I don't think it makes sense to do

```python
from app import db

db = SQLAlchemy()
```
This is probably a problem with my code - do we need the import from app at all?

Reply:
Sorry, haven't cleaned that up. Will remove the import from the app

Reply:
decided to leave it be; still exploring the best approach for importing the db